mesa/space.py

```diff
@@ -1415,6 +1415,13 @@ def get_neighbors(
                             coordinates. i.e. if you are searching for the
                             neighbors of a given agent, True will include that
                             agent in the results.
+
+        Notes:
+            If 1 or more agents are located on pos, include_center=True will remove these agents
+            from the results. So, if you really want to get the neighbors of a given agent,
```

Comment:
I thing I am missing something in this sentence. Based on the change above and my understanding fo the issues shouldnt this sentence read 

If 1 or more agents are located on pos, include_center=False will remove these agents from the results and the given agent (i.e. self). So, if you really want to get the neighbors of a given agent, you should set include_center=True  and then filter the list of agents to remove the given agent (i.e. self).

Reply:
slightly rephrased drawing on your formulation.